{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -0,0 +1,50 @@
+[tox]
+envlist = {py27,py35}-django{18,19}
+
+[pep8]
+max-line-length = 120
+
+[pytest]
+DJANGO_SETTINGS_MODULE = test_settings
+addopts = --cov {{ cookiecutter.app_name }} --cov-report term-missing
+norecursedirs = .* docs requirements
+
+[testenv]
+deps =
+    django18: Django==1.8.14
+    django19: Django==1.9.8
```

Comment:
I agree that a minor incompatibility arbitrarly failing when a Django release happens isn't ideal - it'd be better to bump it and explicitly test. In my experience though, it's best to keep pushing modules to new releases when possible. If not, they end up being pinned to old releases for too long. Anyways, your call...